Debian, Red Hat and others disable font ops by default, so we follow the
same approach to be better with security.

Ref: CVE-2022-45063 (34788211)

--- a/XTerm.ad	2022-11-10 20:15:17.776177946 -0800
+++ b/XTerm.ad	2022-11-10 20:15:25.976118709 -0800
@@ -266,7 +266,7 @@
 !
 ! Depending on your environment, you may wish to disable those by default by
 ! uncommenting one or more of the resource settings below:
-!*allowFontOps: false
+*allowFontOps: false
 !*allowTcapOps: false
 !*allowTitleOps: false
 !*allowWindowOps: false
